diff --git a/drivers/video/msm/mdss/mdss_fb.c b/drivers/video/msm/mdss/mdss_fb.c
index 2fbfe4d..a124445 100644
--- a/drivers/video/msm/mdss/mdss_fb.c
+++ b/drivers/video/msm/mdss/mdss_fb.c
@@ -1667,7 +1667,7 @@ static int mdss_fb_alloc_fbmem_iommu(struct msm_fb_data_type *mfd, int dom)
 					boot_mode_lpm, boot_mode_recovery);
 
 	/* Incase of Normal Booting, Do not reserve FB memory */
-	if ((!boot_mode_lpm) && (!boot_mode_recovery)){
+	if (false) {
 		/* Normal Booting */
 		mfd->fbi->screen_base = NULL;
 		mfd->fbi->fix.smem_start = 0;
diff --git a/arch/arm/mach-msm/memory.c b/arch/arm/mach-msm/memory.c
index fdaec00..e2f1ee3 100644
--- a/arch/arm/mach-msm/memory.c
+++ b/arch/arm/mach-msm/memory.c
@@ -346,7 +346,7 @@ int __init dt_scan_for_memory_reserve(unsigned long node, const char *uname,
 		 * hack to reserve memory for display while booting up in to
 		 * LPM and recovery mode.
 		 */
-		if (boot_mode_lpm || boot_mode_recovery) {
+		if (true) {
 			/*
 			 *  This field is only being used by mdp driver
 			 */
diff --git a/drivers/video/msm/mdss/mdp3_ctrl.c b/drivers/video/msm/mdss/mdp3_ctrl.c
index 71725a1..6f182fe 100644
--- a/drivers/video/msm/mdss/mdp3_ctrl.c
+++ b/drivers/video/msm/mdss/mdp3_ctrl.c
@@ -1174,7 +1174,7 @@ static void mdp3_ctrl_pan_display(struct msm_fb_data_type *mfd)
 
 	mdp3_session->vsync_before_commit = 0;
 #if defined(CONFIG_WHITE_PANEL)
-	if (reset_done && boot_mode_recovery) {
+	if (reset_done) {
 		/*wait for 3 frame time to ensure frame is sent to panel*/
 		msleep(1000 / panel_info->mipi.frame_rate * 3);
 
